integrations/opennms-vmware/src/test/java/org/opennms/protocols/vmware/ServiceInstancePoolTest.java

```diff
@@ -184,7 +186,7 @@ public void verifyHousekeeping() throws Exception {
         s3 = serviceInstancePool.retain("host3.test.de", "username3", "password3");
         Assert.assertEquals("host3.test.de/username3/password3/3/valid", s3.toString());
 
-        Thread.sleep(250);
+        Thread.sleep(350);
```

Comment:
Should we `await().until(...` here instead of fixed sleep times?

Reply:
You are right. I'll fix this.